### ISSUE TITLE
Reword references to PEP 8

### ISSUE THREAD

**Issue by encukou** [coala-analyzer/coala-bears#627]
PEP8Bear gives messages like `The code does not comply to PEP8.`.

It's not really possible to automate checking compliance with PEP 8 – a document, which says, for example, that it's OK to break a guideline if "applying the guideline would make the code less readable".

Instead of "PEP 8", you could for example refer to the tools `autopep8` or `pycodestyle`.

See here for related discussion: https://github.com/PyCQA/pycodestyle/issues/466

**Comment by jayvdb** [comment]
The bear name also needs to be updated.

**Comment by sils1297** [comment]
Yes also we need a decorator like the one for settings for renaming bears
and leaving the deprecated names around.
